assets/js/base/components/radio-control/types.ts

```diff
@@ -14,13 +14,16 @@ export interface RadioControlProps {
 	onChange: ( value: string ) => void;
 	// List of radio control options.
 	options: RadioControlOption[];
+	// Is the control disabled.
+	disabled?: boolean;
```

Comment:
I went and checked, and in this case, it's not erroring for me, but I remember multiple other times where a TS error would be shown if optional props not defined as `boolean | undefined`. For consistency and to prevent errors I think we should change this to `boolean | undefined`, same for below.

The `?` should be enough but we have `exactOptionalPropertyTypes: true` in our tsconfig.

Reply:
In this case we're setting a default of `false` on the incoming props. This means it will never be `undefined`.

Reply:
Even removing that default value doesn't cause the error in places where `disabled` is not passed. I'm not sure what's going on. Happy to keep it as is then, the default value should be fine.

Reply:
👍🏻 Thats the pattern I prefer, otherwise the consuming code may expect you to handle `undefined` everywhere when in reality, its only ever a boolean.